model/task/task_annotations.go

```diff
@@ -36,7 +36,7 @@ func MoveIssueToSuspectedIssue(taskId string, taskExecution int, issue annotatio
 	if err != nil {
 		return errors.Wrapf(err, "finding and modifying task annotation for execution %d of task '%s'", taskExecution, taskId)
 	}
-	if annotation != nil && len(annotation.Issues) == 0 {
```

Comment:
This was giving me a lint error that `annotation` cannot be nil by nature of how it's defined, so it's redundant